user/deployment/pages.md

```diff
@@ -0,0 +1,48 @@
+---
+title: GitHub Pages Deployment
+layout: en
+permalink: /user/deployment/pages/
+---
+
+Travis CI can deploy your static files to [GitHub
+Pages](https://pages.github.com/) after a successful build.
+
+You will need to provide a [personal access
+token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/)
+and set the deployment provider details in `.travis.yml`.
+
+For a minimal configuration, add the following to your `.travis.yml`:
+
+```yaml
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
+  on:
+    branch: master
```

Comment:
Why not add these config options and define them like the "Further Configuration" section? Make it explicit what this 'on' branch is for, plus all other options.